### PR TITLE
(vscode-ember-cli) Added new extension

### PR DESCRIPTION
**Pull request by gep13**

### CODE REVIEW

File: manual/vscode-extensions/vscode-ember-cli/vscode-ember-cli.nuspec [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <projectSourceUrl>https://github.com/felixrieseberg/vsc-ember-cli</projectSourceUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/vscode-extensions/vscode-ember-cli</packageSourceUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/8b94ce90379c08d88333d83aa32f9aa115f5644d/icons/vscode_ember-cli.png</iconUrl>
+    <licenseUrl>https://github.com/felixrieseberg/vsc-ember-cli/blob/master/LICENSE</licenseUrl>
+    <docsUrl>https://github.com/felixrieseberg/vsc-ember-cli/blob/master/README.md</docsUrl>
+    <bugTrackerUrl>https://github.com/felixrieseberg/vsc-ember-cli/issues</bugTrackerUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>Visual Studio Code Ember CLI Extension</summary>
+    <description>Use Ember and Ember Cli directly from Visual Studio Code. After installation, all ember cli commands are available through Code's own command list, enabling you for example to generate blueprints, run tests, or build your app without leaving your editor. Also, IntelliSense for Ember!</description>
+    <tags>Microsoft VisualStudioCode vscode Ember Extension foss</tags>
```
Comment (pascalberger) [inline manual/vscode-extensions/vscode-ember-cli/vscode-ember-cli.nuspec]: Tags in lowercase as per [§1.2.2](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#122-obligatory-tags)

File: manual/vscode-extensions/vscode-ember-cli/vscode-ember-cli.nuspec [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <projectSourceUrl>https://github.com/felixrieseberg/vsc-ember-cli</projectSourceUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/vscode-extensions/vscode-ember-cli</packageSourceUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/8b94ce90379c08d88333d83aa32f9aa115f5644d/icons/vscode_ember-cli.png</iconUrl>
+    <licenseUrl>https://github.com/felixrieseberg/vsc-ember-cli/blob/master/LICENSE</licenseUrl>
+    <docsUrl>https://github.com/felixrieseberg/vsc-ember-cli/blob/master/README.md</docsUrl>
+    <bugTrackerUrl>https://github.com/felixrieseberg/vsc-ember-cli/issues</bugTrackerUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>Visual Studio Code Ember CLI Extension</summary>
+    <description>Use Ember and Ember Cli directly from Visual Studio Code. After installation, all ember cli commands are available through Code's own command list, enabling you for example to generate blueprints, run tests, or build your app without leaving your editor. Also, IntelliSense for Ember!</description>
```
Comment (pascalberger) [inline manual/vscode-extensions/vscode-ember-cli/vscode-ember-cli.nuspec]: Add features section to description as per [§1.2.3](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#123-informative-description) (they have feature description on their marketplace site which you can copy)

File: manual/vscode-extensions/vscode-ember-cli/vscode-ember-cli.nuspec [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>vscode-ember-cli</id>
+    <title>Ember Cli in Visual Studio Code </title>
+    <version>1.0.0</version>
+    <authors>Felix Rieseberg</authors>
+    <owners>gep13 chocolatey</owners>
```
Comment (pascalberger) [inline manual/vscode-extensions/vscode-ember-cli/vscode-ember-cli.nuspec]: Owners should be a comma separated list as per [.nuspec Reference](https://docs.nuget.org/ndocs/schema/nuspec)

**Comment by gep13** [comment]
@pascalberger thank you for the review.  I have implemented all the suggested feedback.

File: manual/vscode-extensions/vscode-ember-cli/vscode-ember-cli.nuspec [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>vscode-ember-cli</id>
+    <title>Ember Cli in Visual Studio Code </title>
+    <version>1.0.0</version>
+    <authors>Felix Rieseberg</authors>
+    <owners>gep13, chocolatey</owners>
```
Comment (pascalberger) [inline manual/vscode-extensions/vscode-ember-cli/vscode-ember-cli.nuspec]: Small one: chocolatey should be listed first according to [§1.2.4](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#124-add-chocolatey-among-owners)

Comment (gep13) [inline manual/vscode-extensions/vscode-ember-cli/vscode-ember-cli.nuspec]: Great catch, done!

**Pull request merged**